README.md

```diff
@@ -113,6 +113,18 @@ python -m build --wheel # in this repository to build the wheel. Assumes you hav
 pip install dist/<generated>.whl
 ```
 
+### Vulkan support
+
+Thanks to [@thewh1teagle](https://github.com/thewh1teagle)
+
+To build and install, clone the repository and run the following commands:
+
+```shell
+export CMAKE_ARGS="-DGGML_VULKAN=1"
+python -m build --wheel # in this repository to build the wheel. Assumes you have installed build with pip install build
+pip install dist/<generated>.whl
```

Comment:
I think this is better, you don't need to export the flag and wait the wheel to build
```shell
GGML_VULKAN=1 pip install . 
```

Or is it not working ?